tests/Unit/Evolution/Systems/GrMhd/ValenciaDivClean/Test_FixConservatives.cpp

```diff
@@ -149,8 +183,19 @@ SPECTRE_TEST_CASE("Unit.Evolution.GrMhd.ValenciaDivClean.FixConservatives",
         0.0,     0.0,
         1.e-12,  0.0,
         enable,  hydro::MagneticFieldTreatment::AssumeNonZero};
+
+    grmhd::ValenciaDivClean::FixConservatives variable_fixer_zero_b_field{
```

Comment:
Assuming clang-tidy is right, can you mark this const?

Reply:
okay